locales/base/translation.json

```
@@ -2115,7 +2115,12 @@
     "sendAmountLabel": "CLAIM LINK AMOUNT",
     "confirmButton": "Send funds and create link",
     "detailsLabel": "HOW IT WORKS",
-    "details": "1. Tap generate link & your funds are sent into the claim link\n2. Send the claim link to a friend\n3. They’ll accept the funds\n4. If funds aren’t claimed, you can reclaim them anytime."
+    "details": "1. Tap generate link & your funds are sent into the claim link\n2. Send the claim link to a friend\n3. They’ll accept the funds\n4. If funds aren’t claimed, you can reclaim them anytime.",
+    "error": {
```

nit: maybe we can be slightly more specific?
```suggestion
    "sendError": {
```